src/Toggle/Object/GetAllToggleStatusRequest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Object;
+
+use Clearbooks\Labs\Client\Toggle\Entity\Group;
+use Clearbooks\Labs\Client\Toggle\Entity\Segment;
+use Clearbooks\Labs\Client\Toggle\Entity\User;
+
+class GetAllToggleStatusRequest
+{
+    /**
+     * @var User
+     */
+    private $user;
+
+    /**
+     * @var Group
+     */
+    private $group;
+
+    /**
+     * @var Segment[]
+     */
+    private $segments;
+
+    /**
+     * @param User $user
+     * @param Group $group
+     * @param Segment[] $segments
+     */
+    public function __construct( User $user, Group $group, array $segments )
```

Comment:
The interactor which uses this request object will return the status for all toggles.
Status includes if the toggle is active or not and if it's locked or not. Currently these are the two statuses we care about in case of toggles on the frontend.